app/components/course_component/course_component.html.erb

```diff
@@ -1,4 +1,7 @@
 <div class="ncce-courses__course">
+  <% if @course.is_new_course %>
+    <%= image_pack_tag('media/images/icons/new-improved-icon.svg', class: "ncce-courses__badge", alt: '') %>
```

Comment:
It would be nicer to have this tag displayed using html & css rather than using a SVG image, so its easier to change in the future, similar to the "this term" tag on the enrichment page, or the "featured" tag on the home page articles


If you stick with SVG image, could you add an alt tag here so screen readers can correctly display them, or if we don't want them to show, instead use aria tags to make it hidden.